src/fillTank.test.js

```diff
@@ -1,11 +1,130 @@
+/* eslint-disable max-len */
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('Should be called', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('Should tank to max capacity, if customer want to much ', () => {
+    const customer = {
+      money: 2085,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 35,
+      },
+    };
+
+    fillTank(customer, 28, 15);
+
+    expect(customer).toEqual({
+      money: 1945,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it('Should tank to max capacity, if customer doesn\'t say amount', () => {
+    const customer = {
+      money: 1500,
+      vehicle: {
+        maxTankCapacity: 29,
+        fuelRemains: 4,
+      },
+    };
+
+    fillTank(customer, 40);
+
+    expect(customer).toEqual({
+      money: 500,
+      vehicle: {
+        maxTankCapacity: 29,
+        fuelRemains: 29,
+      },
+    });
+  });
+
+  it('Should stay at the start position, if it not need to tanking', () => {
+    const customer = {
+      money: 930,
+      vehicle: {
+        maxTankCapacity: 80,
+        fuelRemains: 80,
+      },
+    };
+
+    fillTank(customer, 40);
 
+    expect(customer).toEqual({
+      money: 930,
+      vehicle: {
+        maxTankCapacity: 80,
+        fuelRemains: 80,
+      },
+    });
   });
 
-  // write tests here
+  it('Should tanking possible amount, if customer doest\'n have enough money ', () => {
```

Comment:
The test description 'Should tanking possible amount, if customer doest\'n have enough money' contains a typo. It should be 'doesn't'.